app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/NumericInputViewModel.kt

```diff
@@ -36,25 +36,34 @@ class NumericInputViewModel private constructor(
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           interactionAnswerErrorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
             pendingAnswerError,
-            answerText.isNotEmpty()
+            inputAnswerAvailable = true // Allow blank answer submission
           )
         }
       }
     errorMessage.addOnPropertyChangedCallback(callback)
     isAnswerAvailable.addOnPropertyChangedCallback(callback)
+
+    // Initializing with default values so that submit button is enabled by default.
+    interactionAnswerErrorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
+      pendingAnswerError = null,
+      inputAnswerAvailable = true
+    )
   }
 
-  /** It checks the pending error for the current numeric input, and correspondingly updates the error string based on the specified error category. */
+  /**
+   * It checks the pending error for the current numeric input, and correspondingly
+   * updates the error string based on the specified error category.
+   */
```

Comment:
Can this fit in a single line kdoc? Ok if it can't fit.

Reply:
It will not fit in a single line, but slightly modified it.